### PR TITLE
3/clear empty rte

### PR DESCRIPTION
**Pull request by abea**
Resolves https://apostrophecms.atlassian.net/browse/A30U-682, "If I had empty richtext while in Edit mode, they are removed on publish/save"

### CODE REVIEW

File: modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue [hunk 1]
```diff
@@ -201,6 +201,7 @@ export default {
         busy: true
       });
       this.patches = [];
+      await this.refresh();
```
Comment (boutell) [inline modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue:204]: This is going to refresh constantly with autosave.

**Review by boutell — CHANGES_REQUESTED** [review]
Now that I see how this works out, if we're going to do this I think we need to do it on the front end. Otherwise the front end doesn't know what the backend is doing which leads to problems for undo/redo and the integrity of the patching algorithm. Also on the front end we can address what happens with autosave if you add a rich text widget and just don't type right away... blam, it's gone!

You see why I was really leery of going ahead on it, but it's a useful experiment and first step.

**Comment by boutell** [comment]
I would recommend not taking this further until autosave lands for the sake of not having to rework it twice.

**Comment by abea** [comment]
@boutell We can wait, but the idea is that this wouldn't be done until the user's intentional save point. That wouldn't be during auto-save. Do we have those anymore?

**Comment by boutell** [comment]
We don't have them at the *moment*, but we will have them when draft/publish becomes a thing.

At that point we could make sure a refresh happens, that's a reasonable thing to do at such a moment yep.

**Comment by boutell** [comment]
Closing this in its current form as discussed. We talked about various options for the future, including making it easier to spot an empty (or intentional placeholder) rich text widget as an editor, or doing this at specific times in the life cycle that wouldn't cause issues for contextual editing. I also suggested doing it browser-side, but the trouble with that is I don't see any really good time for it. If I just added a rich text widget, I probably do not want it to go away at least half of the time even if I immediately go on to making another.

**Pull request closed without merging**